src/worldfactory.cpp

```diff
@@ -1251,9 +1252,13 @@ void worldfactory::draw_modselection_borders( const catacurses::window &win, con
 
     // Add tips & hints
     fold_and_print(win, FULL_SCREEN_HEIGHT - 7, 2, getmaxx(win) - 4, c_green,
-                   _("Press %s to save the list of active mods as default. Press %s for help."),
+                   _("Press %s = Save Load Order as default.  %s = Controls %s/%s= Prev/Next Option. %s/%s = Prev/Next Tab."),
```

Comment:
"Press X = " sounds weird. Old "Press X to ~win~ save..." was better. Or remove the "Press" part.
Also, decapitalize all text. 
Also, add a space in `%s/%s=`.

Reply:
Good point there.  And yeah, I wasn't sure if %s/%s or %s / %s was better.  Changes made.

Reply:
Uhmm, sorry for not being clear enough. I mean add space between `%s/%s` and `=`. I liked the old way (`%s/%s`) by the way.